az-aks-ssh.sh

```diff
@@ -72,9 +78,32 @@ while [[ $# -gt 0 ]]; do
             CLEANUP="yes"
             shift
             ;;
+        -h|--help)
+            usage
+            ;;
     esac
 done
 
+# Try to infer some settings from the environment as a convenience.
+
+if [ -z "${RESOURCE_GROUP:-}" ]; then
```

Comment:
BTW, `$NODE_NAME` had this same issue (stems from the use of `set -eu` up top, which is a good thing), so I added a default to `any` for it as well so that the bare command execute will get you a shell somewhere if your environment variables are already set.
Alternatively, could just throw an error if `NODE_NAME` is left unspecified.